README.md

```diff
@@ -254,7 +254,7 @@ taken from [bat](https://github.com/sharkdp/bat/blob/master/assets/patches/Markd
 
 ## Configuration
 The default config TOML file is located in
-* Linux: `/home/<username>/.config`
+* Linux: `/home/<username>/.config/the-way`
 * Mac: `/Users/<username>/Library/Preferences`
```

Comment:
Is there an additional directory on MacOS as well? Would be good to keep the docs consistent.

Reply:
Yes, it's called "rs.the-way" in MacOS

Reply:
@jgarte Could you add "/rs.the-way" to the end of the MacOS path?

Reply:
@Ninjani done! 👍🏿